polyfill.js

```diff
@@ -37,5 +37,5 @@ if (typeof Promise.prototype.finally !== 'function') {
 			return newPromise;
 		}
 	};
-	Promise.prototype.finally = shim.finally;
```

Comment:
Wouldn't the test fail anyway in ES3 browsers since it requires `Promise` and arrow functions?